src/nvim/syntax.c

```diff
@@ -5785,6 +5785,8 @@ static char *(highlight_init_both[]) =
       "TabLineSel term=bold cterm=bold gui=bold"),
   CENT("TabLineFill term=reverse cterm=reverse",
       "TabLineFill term=reverse cterm=reverse gui=reverse"),
+  CENT("TermCursor term=reverse cterm=reverse",
```

Comment:
you can remove the `term=reverse` attribute, it's ignored and only accepted to avoid errors on existing color schemes